### ISSUE TITLE
Any plans to upgrade the Ubuntu version on the base images?

### ISSUE THREAD

**Issue by uberhacker** [ciandt-dev/Docker4Acquia#21]
Support for Ubuntu 12.04 (aka precise) ends this month.  Should the images be upgraded to 16.04 (aka xenial) to match the current cloud environment?

**Comment by ivan-pinatti** [comment]
Yes, we do have plans do upgrade the images to match Acquia environment.
Since we try to mimic it we have to wait to get access to the new versions and do the changes in order to have every single piece of configuration alike.
